test/org/sagebionetworks/bridge/TestUserAdminHelper.java

```diff
@@ -46,17 +46,25 @@
         private final Study study;
         private final UserSession session;
 
-        public TestUser(SignUp signUp, Study study, UserSession session) {
-            this.email = signUp.getEmail();
-            this.password = signUp.getPassword();
-            this.roles = Sets.newHashSet(signUp.getRoles());
+        public TestUser(String email, String password, Set<Roles> roles, Set<String> dataGroups, Study study,
+                UserSession session) {
+            this.email = email;
+            this.password = password;
+            this.roles = Sets.newHashSet(TEST_USERS);
+            if (roles != null) {
+                this.roles.addAll(roles);    
+            }
             this.roles.add(TEST_USERS);
```

Comment:
Minor: You create a set with TEST_USERS, add a bunch of roles, and then add TEST_USER.